test/stat_tracker_test.rb

```diff
@@ -9,17 +9,20 @@ def setup
     team_path = './fixtures/teams_fixture.csv'
     game_teams_path = './fixtures/game_teams_fixture.csv'
 
-    locations = {
+    file_path_locations = {
```

Comment:
We changed the name of this hash from "locations" to "file_path_locations" because it's more descriptive and made more sense because we can see what the "location" is.

Reply:
Agreed, this helps us distinguish what we are doing.  Re: Paired review w/ AJ.